control-plane-operator/hostedclusterconfigoperator/controllers/hcpstatus/hcpstatus.go

```diff
@@ -149,6 +149,12 @@ func (h *hcpStatusReconciler) reconcile(ctx context.Context, hcp *hyperv1.Hosted
 			}
 		}
 
+		// If CVO has no Upgradeable condition, consider the HCP upgradable according to the CVO
```

Comment:
wouldn't this also enter this path if the cvo is reporting upgradable unknown? is that ok or should we check if cvoConditions[conditionType] == nil here?

Reply:
Yes, it would.  I would probably be better to only do the path if the condition was not found.

My understanding is that if the CVO has an Upgradeable condition, the state is False with a reason.  If that is the case, this works, but it might not always work.